classes/install/Installer.php

```diff
@@ -854,17 +854,12 @@ public function installFilterConfig($filterConfigFile)
     /**
      * Check to see whether a column exists.
      * Used in installer XML in conditional checks on <data> nodes.
-     *
-     * @param string $tableName
-     * @param string $columnName
-     *
-     * @return bool
      */
-    public function columnExists($tableName, $columnName)
+    public function columnExists(string $tableName, string $columnName): bool
```

Comment:
I think `Schema::hasColumn` could be used directly in cases where this is required, so this can be removed.